Pushed upstream at https://bugs.launchpad.net/alarmclock/+bug/395979

	* src/Makefile.am: Use the libtool flag '-export-dynamic' in order to get
	the correct linker flag on all platforms.

--- origsrc/alarm-clock-1.1/src/Makefile.am	2009-07-05 09:25:39.000000000 -0500
+++ src/alarm-clock-1.1/src/Makefile.am	2009-07-05 23:15:13.441782400 -0500
@@ -46,7 +46,7 @@ alarmclock_SOURCES = \
 	export.c      
 
 alarmclock_LDFLAGS = \
-	-Wl,--export-dynamic
+	-export-dynamic
 
 alarmclock_LDADD = $(ALARMCLOCK_LIBS)
 
